package/MDAnalysis/analysis/density.py

```diff
@@ -208,12 +211,16 @@ class DensityAnalysis(AnalysisBase):
     zdim : float (optional)
             User defined z dimension box edge in ångström.
 
-    Returns
-    -------
-    :class:`Density`
+    Attributes
+    ----------
+    results.density : :class:`Density`
             A :class:`Density` instance containing a physical density of units
             :math:`Angstrom^{-3}`.
 
+    density : :class:`Density`
```

Comment:
You can put the deprecated reST directly here. Not sure if we have a policy that says it must go at the end. It would make sense to have it here.